CMakeLists.txt

```diff
@@ -58,7 +58,6 @@ tvm_option(USE_OPENMP "Build with OpenMP thread pool implementation" OFF)
 tvm_option(USE_RELAY_DEBUG "Building Relay in debug mode..." OFF)
 tvm_option(USE_RTTI "Build with RTTI" ON)
 tvm_option(USE_MSVC_MT "Build with MT" OFF)
-tvm_option(USE_MICRO "Build with Micro TVM support" OFF)
```

Comment:
Too keep the `USE_MICRO` option, we can have it in the CMakelist and make it default ON. Then we can remove it from `cmake/config.cmake` file

Reply:
thanks, done.